hastexo/hastexo.py

```diff
@@ -252,11 +257,11 @@ def parse_xml(cls, node, runtime, keys, id_generator):
 
                     elif child.tag == "hook_events":
                         hook_events = {
-                            "suspend": bool(
+                            "suspend": cls.to_bool(
```

Comment:
... or else, drop the wrapper method and just open-code `bool(strtobool(<value>))` here.

Reply:
Updated the PR with this.